### ISSUE TITLE
[New Rule] Use list of strings instead of splitting a constant string

### ISSUE THREAD

**Issue by MartinThoma** [MartinThoma/flake8-simplify#86]
## Explanation

You want a list of strings? Create one directly:

* A list of strings can be formatted by black.
* Your editor can sort the lines
* You don't have to worry about how to keep trailing / leading spaces or how to add newlines to the strings

## Example

```python
# Bad
domains = "de com net org".split()

# Good
domains = ["de", "com", "net", "org"]
```

**Comment by MartinThoma** [comment]
```
$ astpretty --no-show-offsets /dev/stdin <<< `cat example.txt`
Module(
    body=[
        Assign(
            targets=[Name(id='domains', ctx=Store())],
            value=Call(
                func=Attribute(
                    value=Constant(value='de com net org', kind=None),
                    attr='split',
                    ctx=Load(),
                ),
                args=[],
                keywords=[],
            ),
            type_comment=None,
        ),
    ],
    type_ignores=[],
)
```

**Comment by MartinThoma** [comment]
The relevant part is:

```
Call(
                func=Attribute(
                    value=Constant(value='de com net org', kind=None),
                    attr='split',
                    ctx=Load(),
                )
```
